app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,7 @@
 <resources>
     <string name="app_name">Constitution</string>
+    <string name="qaraqalpaqsha">Qaraqalpaqsha</string>
+    <string name="orissha">Русский</string>
+    <string name="ózbekshe">O\'zbekcha</string>
```

Comment:
Do not use unicode or unusual symbols as names (only latin letters are allowed). Also, try to be consistent, if all other namings are in English, use English. Here we can have something like `lang_*` (lang_qq, lang_ru,...)